flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Utils/NXSystemMessages.js

```diff
@@ -73,5 +73,5 @@ const NXFictionalMessages = {
     noWptInfos:             new McduMessage("NO WAYPOINT INFOS", false, false),
     emptyMessage:           new McduMessage(""),
     weightUnitChanged:      new McduMessage("UNIT CHANGED RELOAD A/C", true, true),
-    resetMcdu:              new McduMessage("RESET MCDU TO APPLY", true, false)
+    reloadPlane:             new McduMessage("RELOAD PLANE", true, false)
```

Comment:
Nitpick: can we make it RELOAD A/C, or RELOAD AIRCRAFT, to match the message immediately above?

Reply:
Sure, should be an easy change